geckomat/utilities/FVA/comparativeFVA.m

```diff
@@ -0,0 +1,138 @@
+function [FVA_Dists,indexes,stats] = comparativeFVA(model,ecModel,c_source,chemostat,tol,blockedMets)
+% comparativeFVA
+%  
+% This function goes through each of the rxns in a metabolic model and
+% gets its flux variability range, then the rxn is mapped into an EC
+% version of it to perform the correspondent variability analysis and
+% finally compares and plots the cumulative flux variability distributions. 
+%
+%   model       MATLAB GEM structure, constrained with the desired culture
```

Comment:
From the code I think the `model` input has to come in irreversible format, however this is not stated in the documentation. Please clarify this.